### DISCUSSION TITLE
python-duckdb compatibility with pyarrow

### DISCUSSION THREAD

**Discussion by xuyan-ru** [duckdb/duckdb#8689] [Q&A] [answered]
Hi community, 

I am new here, just have a quick question regarding duckdb together with pyarrow. When trying to install python-duckdb using conda, it's always trying to downgrade my pyarrow from 12.0.0 to 9.0.0, which is big, before really doing this, I'd like to ask is there any incompatibility between duckdb with pyarrow beyond 9.0.0? 

Some of my environment info:
conda: 23.7.3
python: 3.11.4

Thanks!

**Comment by Mause** [comment, marked as answer]
Hey there, we don't maintain the conda feedstock, feel free to raise an issue here instead: https://github.com/conda-forge/python-duckdb-feedstock
